_sass/color_schemes/dark.scss

```diff
@@ -12,21 +12,7 @@ $search-background-color: $grey-dk-250;
 $table-background-color: $grey-dk-250;
 $feedback-color: darken($sidebar-color, 3%);
 
-// The following highlight theme is more legible than that used for the light color scheme
-
-// @import "./vendor/OneDarkJekyll/syntax-one-dark";
-// $code-background-color: #282c34; // OneDarkJekyll default for syntax-one-dark
-// $code-linenumber-color: #abb2bf; // OneDarkJekyll .nf for syntax-one-dark
-
-@import "./vendor/OneDarkJekyll/syntax-one-dark-vivid";
+@import "./vendor/OneDarkJekyll/syntax";
```

Comment:
I suggest to comment here that syntax.scss is actually a copy of OneDarkJekyll/syntax-one-dark-vivid.scss. The comments referring to syntax-one-dark-vivid on the code color rules confused me.